packages/core/src/Stores/client-store.js

```diff
@@ -2501,34 +2507,56 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    responseCTraderTradingPlatformAvailableAccounts(response) {
+        if (!response.error) {
+            this.ctrader_trading_platform_available_accounts = response.trading_platform_available_accounts;
+        }
+    }
+
     responseTradingPlatformAccountsList(response) {
         const { platform } = response.echo_req || {};
 
         this[`is_populating_${platform}_account_list`] = false;
         this[`${platform}_accounts_list_error`] = null;
 
         if (!response.error) {
-            this[`${platform}_accounts_list`] = response.trading_platform_accounts.map(account => {
-                const display_login = account.error ? account.error.details.account_id : account.account_id;
-                if (account.error) {
-                    const { account_type, server } = account.error.details;
-                    if (platform === CFD_PLATFORMS.DXTRADE || platform === CFD_PLATFORMS.CTRADER) {
-                        this.setCFDDisabledSignupTypes(platform, {
-                            [account_type]: true,
-                        });
+            this[`${platform}_accounts_list`] = response.trading_platform_accounts
+                .slice()
+                .sort((a, b) => {
+                    const loginA = a?.login;
+                    const loginB = b?.login;
+
+                    if (loginA && loginB) {
+                        if (loginA < loginB) {
+                            return -1;
+                        }
+                        if (loginA > loginB) {
+                            return 1;
+                        }
+                    }
+                    return 0;
+                })
```

Comment:
Can you make use of a hook instead of keeping these calculations in store?

Reply:
I will have a discussion with @sara-fs tomorrow to get the sorted data from the BE side since we need to sort the data this way in 2 to 3 stores in the legacy code.

Reply:
The new change is pushed